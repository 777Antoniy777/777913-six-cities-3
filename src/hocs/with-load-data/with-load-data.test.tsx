import React from "react";
import renderer from "react-test-renderer";
import withLoadData from "./with-load-data";

// set mocha data
const props = [];

const getData = () => {};

const MockComponent = () => {
  return (
    <div></div>
  );
};

const ComponentWrappedHOC = withLoadData(MockComponent);

it(`render withLoadData`, () => {
  const tree = renderer.create((
    <ComponentWrappedHOC
      props={props}
      getData={getData}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
