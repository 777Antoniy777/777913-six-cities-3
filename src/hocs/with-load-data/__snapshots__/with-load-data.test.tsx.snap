// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withLoadData 1`] = `<div />`;
